dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/AcmeValidityAndRenewTest.java

```diff
@@ -0,0 +1,373 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.acme.fat;
+
+import static junit.framework.Assert.assertNotNull;
+import static org.junit.Assert.assertNotSame;
+
+import java.security.cert.Certificate;
+import java.security.cert.X509Certificate;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.acme.docker.CAContainer;
+import com.ibm.ws.security.acme.internal.util.AcmeConstants;
+import com.ibm.ws.security.acme.utils.AcmeFatUtils;
+
+import componenttest.annotation.CheckForLeakedPasswords;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Verify that we log the correct warnings for different
+ * validity/renewBeforeExpiration settings
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class AcmeValidityAndRenewTest {
+
+	@Server("com.ibm.ws.security.acme.fat.simple")
+	public static LibertyServer server;
+
+	private static ServerConfiguration ORIGINAL_CONFIG;
+	private static final String[] DOMAINS1 = { "domain1.com" };
+
+	public static CAContainer caContainer;
+
+	public static final long timeBufferToExpire = 30000L; // milliseconds
+
+	@BeforeClass
+	public static void beforeClass() throws Exception {
+		ORIGINAL_CONFIG = server.getServerConfiguration();
+		caContainer = new com.ibm.ws.security.acme.docker.pebble.PebbleContainer();
+		AcmeFatUtils.checkPortOpen(caContainer.getHttpPort(), 60000);
+	}
+
+	@AfterClass
+	public static void afterClass() throws Exception {
+		if (caContainer != null) {
+			caContainer.stop();
+		}
+	}
+
+	@After
+	public void afterTest() {
+		/*
+		 * Cleanup any generated ACME files.
+		 */
+		AcmeFatUtils.deleteAcmeFiles(server);
+
+	}
+
+	/**
+	 * Whether to use "acme://" style URIs, or "https://" style URIs.
+	 * 
+	 * <p/>
+	 * Sub-classes can override this method to determine which style URIs should be
+	 * used.
+	 * 
+	 * @return Whether to use "acme://" style URIs.
+	 */
+	protected boolean useAcmeURIs() {
```

Comment:
Probably don't need this. We used this in AcmeSimpleTest so we could make it work for both Pebble (https and acme URIs) and Boulder.

Reply:
Removed